src/main/java/com/arpnetworking/metrics/mad/model/statistics/HistogramStatistic.java

```diff
@@ -250,29 +252,79 @@ public void recordValue(final double value) {
             recordValue(value, 1);
         }
 
+        /**
+         * Records a packed value into the histogram. The packed values must be at the
+         * precision that this {@link Histogram} was declared with!
```

Comment:
It would be truncated into a different bucket and the error of the histogram will be undefined. Merging histograms with different precisions is something we'll need to do when we actually add variable precision support to the flow. Effectively, it will involve downsampling all the buckets to the lowest precision in a given histogram.

^ FYI @BrandonArp